ivy/functional/ivy/random.py

```diff
@@ -538,10 +538,39 @@ def shuffle(
 
     Examples
     --------
+    With :class:`ivy.Array` input:
+
     >>> x = ivy.array([1, 2, 3, 4, 5])
     >>> y = ivy.shuffle(x)
     >>> print(y)
     ivy.array([2, 1, 4, 3, 5])
 
+    >>> x = ivy.array([1,0,5])
+    >>> y = ivy.array([0, 0, 0])
+    >>> z = ivy.shuffle(x, out=y)
```

Comment:
no need to put `z = ` here, the line can just be `>>> ivy.shuffle(x, out=y)`

Reply:
Additionally, could add a value for the `seed` parameter here as it is required to show each parameter being used at least once

Reply:
Done, fixed this. Also, added a separate example for `seed` argument.